build/profiles/freenas/packages/freenasUI/config

```diff
@@ -21,14 +21,15 @@ include = /usr/local/www/freenasUI
 # Fresh installs don't need this.
 # However, if the package name changed, we would.
 
-#post-install = 
+post-install = 
 #	if [ -f /data/freenas-v1.db ]; then
 #	   /usr/bin/touch /data/cd-upgrade
 #	   /usr/bin/touch /data/need-update
 #	fi
 #	test -f /data/freenas-v1.db && \
 #		/usr/bin/yes | /usr/local/bin/python \
 #		            /usr/local/www/freenasUI/manage.py migrate --all --merge --delete-ghost-migrations
+	/usr/local/bin/sqlite3 /data/freenas-v1.db "REPLACE INTO system_keyvalue (key, value) VALUES ('fake_migration', 'true')"
```

Comment:
Can we do the install using the database migration?

So this will also work with 12.

Reply:
FREENAS_INSTALL env var is set while running the migrations on install time

Reply:
> Can we do the install using the database migration?

It will also run on upgrade, won't it? Which we don't want.